src/utils/calldata/cairo.ts

```diff
@@ -203,14 +203,16 @@ export function getAbiContractVersion(abi: Abi): ContractVersion {
 
   // determine by function io types "Cairo 1.1" or "Cairo 0.0"
   // find first function with inputs or outputs
-  const testFunction = abi.find(
-    (it) => it.type === 'function' && (it.inputs.length || it.outputs.length)
+  const testSubject = abi.find(
+    (it) =>
+      (it.type === 'function' || it.type === 'constructor') &&
```

Comment:
no need to do it now, but later we can extract these comparisons somewhere to not have magic strings, since we have these same comparisons in a couple of more files cc @penovicp